packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -117,10 +117,11 @@ describe('Deposit ERC20 Token', () => {
           .click()
           .then(() => {
             cy.confirmMetamaskTransaction().then(() => {
+              cy.findByText('~10 mins remaining').should('be.visible')
               cy.findByText(
-                `Moving ${formatAmount(ERC20AmountToSend, {
+                `${formatAmount(0.0001, {
```

Comment:
Applies for all - can this text check be a bit more specific to avoid false-positives? There can be multiple transactions in the table with `0.0001 WETH`.
Something like traversing the DOM and finding this element and then doing a text check?